package.json

```diff
@@ -110,6 +111,7 @@
     "patch-package": "^7.0.0",
     "react-addons-test-utils": "^15.6.2",
     "ts-jest": "^29.1.1",
+    "ts-node": "^10.9.2",
```

Comment:
Is this used directly by our scripts? I couldn't find find it elsewhere in this diff

Reply:
By default, Forge is unable to load `forge.config.ts` files unless `ts-node` is added because it's a `.ts` module loader that gets added via https://www.npmjs.com/package/rechoir.

This wasn't documented previously, but I've added docs here (in review): https://www.electronforge.io/~/changes/RAnueuhc9hRBiJm9n8cC/config/typescript-configuration